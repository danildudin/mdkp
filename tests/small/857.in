4 2
29 97 72 89 
30 54 76 68 98
32 92 17 23 564
