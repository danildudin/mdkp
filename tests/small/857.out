126
2
0 1 
